### DISCUSSION TITLE
Dashed Line Spec?

### DISCUSSION THREAD

**Discussion by joseandroidengineer** [patrykandpatrick/vico#798] [Questions] [answered]
### Question

Loving this library, wanted to ask is there a way we can make the CartesianLine chart to have a dashed line instead of just a solid one? If not maybe we can add support for it in the future?

### Vico version(s)

2.0.0 alpha.21

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hi @joseandroidengineer, this is a planned addition, as mentioned in #469. We want to look into it shortly.

**Reply by joseandroidengineer** [reply]
Thank you for the swift response! Much appreciated
